org.lflang/src/org/lflang/generator/c/CCompiler.java

```diff
@@ -275,6 +286,33 @@ public LFCommand buildCmakeCommand() {
         return command;
     }
 
+    /**
+     * Return a flash/emulate command using west.
+     * If board is null (defaults to qemu_cortex_m3) or qemu_*
+     * Return a flash command which runs the target as an emulation
+     * If ordinary target, return `west flash`
+     */
+    public LFCommand buildWestFlashCommand() {
+        // Set the build directory to be "build"
+        Path buildPath = fileConfig.getSrcGenPath().resolve("build");
+        String board = targetConfig.platformOptions.board;
+        LFCommand cmd;
+        if (board == null || board.startsWith("qemu")) {
```

Comment:
There might be a way to change the type of `board` back to an enum (as it was before this PR), and thereby to avoid having to do this sort of string-matching -- or at least, to restrict the string-matching to a single place that is not buried deep in `org.lflang.generator.c`. Enums can be constructed with parameters, so even though the total number of boards might be too large to enumerate, you could have a few types of boards (e.g., `ARDUINO`, `QEMU`, ...) that also come with a `String` that specifies the exact name of the board.

Reply:
Good point. @arengarajan99 and I will have to figure out something clever here. Or wait until it is all factored out into the package manager...